packages/lesmis/src/api/api.js

```diff
@@ -58,6 +58,11 @@ const request = async (endpoint, options) => {
     if (error.response) {
       const { data } = error.response;
 
+      // logout the user out if we get a 401 response
+      if (error.response.status === 401 || data.code === 401) {
+        request('logout', { method: 'post' });
```

Comment:
Should we `return` after this, to avoid the error handling below?

Reply:
hmm good question. I'm not entirely sure either. I did a quick test and in all the cases I can think of it makes no difference. I'm inclined to not return as it is more predictable to keep the consistent flow of error handling?!

Reply:
Sure thing!